modules/i18n/src/models/common.ts

```diff
@@ -63,12 +63,18 @@ export interface NotificationItem {
 }
 
 /**
- * Model for confirmation boxes.
+ * Model for info boxes.
  */
-export interface Confirmation {
+export interface InfoModal {
```

Comment:
I think we can keep this generic because irrespective of the type, our all modals follow this content format. WDYT?

```suggestion
export interface Modal {
```